app/controllers/api/v1/orders_controller.rb

```diff
@@ -0,0 +1,72 @@
+class Api::V1::OrdersController < Api::V1::ApplicationController
+  before_action :authenticate, only: [:add_to_order, :remove_from_order]
+  include UserAuthorization
+
+  def show
+    @order = current_user.orders.find(params[:id])
```

Comment:
same with the global variables. you should do a double check on all api controllers